packages/components/src/z-stack/styles.ts

```diff
@@ -19,6 +19,9 @@ export const ZStackChildView = styled.div< {
 	offsetAmount: number;
 	zIndex: number;
 } >`
+	/* TODO: split this change to a separate PR */
```

Comment:
Should this comment be removed, or is the PR still a work in progress?

Reply:
For now, I've kept the changes to the `ZStack` component this PR for ease of review, but I plan on applying it in a separate PR.

I will proceed to delete those lines just before merging

Reply:
With the latest changes, the `ZStack` component doesn't need to be tweaked anymore.